src/windshaft/filters/base.js

```diff
@@ -1,12 +1,37 @@
 var Model = require('cdb/core/model');
 
-module.exports = Model.extend({
 
+/**
+ * @classdesc Abstract base class for all filters. **Do not use directly**
+ *
+ * Any filter must implement at
+ * least the operations `isEmpty()` and `toJSON()`.
+ *
+ * @class
+ * @extends Model
+ */
+var FilterBase = Model.extend( /** @lends FilterBase.prototype */ {
```

Comment:
As can be seen in the [screenshots comparing the output](https://github.com/CartoDB/cartodb.js/pull/849#discussion_r45630364), neither tool resolves namespace (directory path or assignment in the bundle entry file, I.e. `cdb.windshaft.filters.FilterBase = require('./windshaft/filters/base');`. I think this should be resolved before merging since one file/class name might be reused in various context (e.g. the widget models…)